src/llfs/slot_writer.test.cpp

```diff
@@ -0,0 +1,45 @@
+//#=##=##=#==#=#==#===#+==#+==========+==+=+=+=+=+=++=+++=+++++=-++++=-+++++++++++
+//
+// Part of the LLFS Project, under Apache License v2.0.
+// See https://www.apache.org/licenses/LICENSE-2.0 for license information.
+// SPDX short identifier: Apache-2.0
+//
+//+++++++++++-+-+--+----- --- -- -  -  -   -
+
+#include <llfs/slot_writer.hpp>
```

Comment:
Important point: this isn't a typo; we intentionally include the .hpp file twice at the top of the corresponding .test.cpp file to test that it doesn't break anything to double-include the file (this is perfectly valid; for example, it can be included through two different indirect chains of includes).  Basically this is a compile-time test for the presence of `#pragma once` and/or the double-inclusion guard macros.